wolfi-base.yaml

```diff
@@ -26,18 +26,16 @@ test:
     - name: "Commands exist"
       runs: |
         for x in apk sh ; do
-          if ! which $x ; then
-            echo "$x not found"
-            exit 1
-          fi
+          which $x
+          type $x
+          command -v $x
         done
     - name: "Commands do not exist"
       runs: |
-        for x in curl ; do
-          if which $x ; then
-            echo "$x not found"
-            exit 1
-          fi
+        for x in curl git wget; do
+          which $x && echo "$x found but should not be installed" && exit 1
+          type $x && echo "$x found but should not be installed" && exit 1
+          command -v $x && echo "$x found but should not be installed" && exit 1
```

Comment:
Are you sure this does what you want?  Unfortunately these won't run presubmit unless there is an epoch bump

Reply:
I reworked these, as a nested loop, with more clear error messages and exit criteria...see what you think in the newer push....